### DISCUSSION TITLE
Can request be imported?

### DISCUSSION THREAD

**Discussion by krh1009** [sct/overseerr#867] [Q&A] [answered]
I'm thinking of switching to Overseerr from Ombi.  Is there a way to import old request from Ombi into OverSeerr?

Use Case: Last week Tom requested the movie XYZ using Ombi. Today I turn off Ombi and turn on Overseerr, tomorrow the movie XYZ gets downloaded. Tom will not get an email since Ombi is off and Overseerr doesn't know Tom requested XYZ.

**Comment by TheCatLady** [comment, marked as answer]
Hi @krh1009!  Unfortunately, we do not support importing data from Ombi at this time.  However, we do have an awesome "Request As" feature that allows admins to submit requests on behalf of other users.  So, in your hypothetical scenario, you could submit a request for movie XYZ on behalf of Tom, so that he will receive a notification when it becomes available.

**Comment by krh1009** [comment]
Great work around !  Thank you. 

(sent from my iPhone)

**Reply by TheCatLady** [reply]
No problem!  If you need additional help, I recommend joining our [Discord server](https://discord.gg/PkCWJSeCk7).  We have a pretty active (and growing!) community, and you're likely to get a quicker response there.
